tests/import_proxy_test.go

```diff
@@ -359,3 +369,24 @@ func cleanClusterWideProxy(ocpClient *configclient.Clientset, clusterWideProxySp
 		return false
 	}, timeout, pollingInterval).Should(BeTrue())
 }
+
+func manipulateMachineConfigPools(mcfgClient *mcfgclient.Clientset, paused bool) {
+	workerMCP, err := mcfgClient.MachineconfigurationV1().MachineConfigPools().Get(context.TODO(), "worker", metav1.GetOptions{})
+	Expect(err).ToNot(HaveOccurred())
+	masterMCP, err := mcfgClient.MachineconfigurationV1().MachineConfigPools().Get(context.TODO(), "master", metav1.GetOptions{})
```

Comment:
IIUC there are discussions to rename `master` MCP to `control-plane` due to the concerns with non inclusive language.
Maybe we should loop over all MCP instead of hardcoding their names.